OpenID_Connect/oidc_config.sh

```diff
@@ -37,6 +38,123 @@ function print_usage() {
   exit 1
 }
 
+function get_settings_id() {
+  SETTINGS_ID=`curl $CURL_OPTS \
+    -H "Authorization: Bearer $API_TOKEN" \
+    -X GET \
+    ${SETTINGS_ENDPOINT} | jq '.authenticationSettings | .[] | select(.type=="'"${SSO_KEYWORD}"'") | .id'`
+}
+
+function exit_if_no_settings_id() {
+  if [ -z "$SETTINGS_ID" ] ; then
+    echo "No ${SSO_KEYWORD} settings are set"
+    echo
+    exit 0
+  fi
+}
+
+function get_active_settings_type() {
+  ACTIVE_SETTINGS_TYPE=`curl $CURL_OPTS \
+    -H "Authorization: Bearer $API_TOKEN" \
+    -X GET \
+    $ACTIVE_ENDPOINT | jq '.activeSettings | .type'`
+}
+
+function set_provider_variables() {
```

Comment:
should this be renamed to `check_provider_variables`?

Reply:
You are right. This came for an ugly copy-paste from SAML file.